argo-cd-apps/overlays/konflux-public-staging/delete-applications.yaml

```diff
@@ -11,3 +11,9 @@ kind: ApplicationSet
 metadata:
   name: konflux-rbac
 $patch: delete
+---
```

Comment:
you should remove this change in order to deploy to staging.

Reply:
great catch!

Reply:
Fixed in 1cd4b0af.